src/configure.ts

```diff
@@ -88,46 +88,49 @@ export async function jestPreviewConfigure(
   }
 }
 
-function autoRunPreview() {
+// Omit only, skip, todo, concurrent, each. Couldn't use Omit. Just redeclare for simplicity
+interface RawIt {
+  (name: string, fn?: jest.ProvidesCallback, timeout?: number): void;
+}
```

Comment:
How about this? Reference: https://stackoverflow.com/questions/55376089/extract-call-signature-from-a-type

```ts
type RawIt = (...args: Parameters<jest.It>) => ReturnType<jest.It>
```

Reply:
@thanhsonng It's an excellent idea. I will commit that. Thanks.
(You can use GitHub's suggestion feature, so you can co-author the suggestion code. It would ease the tragedy of squashing your commits in the past.)

Reply:
thanks for reminding me of that :)